n6001/hardware/ofs_n6001/build/scripts/run.sh

```diff
@@ -103,72 +108,20 @@ if [ -n "$OFS_ASP_ENV_ENABLE_ASE" ]; then
     exit $?
 fi
 
-echo "Calling compile_script"
-quartus_sh -t scripts/compile_script.tcl "$BSP_FLOW"
```

Comment:
compile-script isn't needed anymore, so it is deleted from the repo. Instead of calling out each individual step we just call a single compile command. The single command is needed in order to leverage all of the tcl scripts included in pr-build-template (including the user-clock and gen-gbs stuff).